node/silkworm/stagedsync/stage_execution.cpp

```diff
@@ -21,10 +21,10 @@
 #include <silkworm/common/endian.hpp>
 #include <silkworm/common/log.hpp>
 #include <silkworm/common/stopwatch.hpp>
+#include <silkworm/consensus/ethash/ethash.hpp>
```

Comment:
Probably needs consensus_engine.hpp instead of ethash.hpp specifically.

Reply:
Right.